### ISSUE TITLE
obs portable stoill used C:\ProgramData

### ISSUE THREAD

**Issue by 1234G-G4321** [obsproject/obs-studio#10486]
### Operating System Info

Windows 10

### OBS Studio Version

30.1.1

### OBS Studio Log URL

https://obsproject.com/logs/8eBLXexJZ6QihpCT

### Expected Behavior

portable version should only use obs directory instead using system folder.

### Current Behavior

obs portable still use C:\ProgramData folder

auto create two folders (even using portable)
C:\ProgramData\obs-studio
C:\ProgramData\obs-studio-hook

### Steps to Reproduce

1. open obs portable (obs_portable_mode.txt method)
2. see C:\ProgramData folder
3. two folders created seen
...

**Comment by RytoEX** [comment]
Running OBS Studio 30.1.2 in Portable Mode in a fresh Windows Sandbox, I do not see exactly this behavior. Instead, I get the following folders:
```
C:\ProgramData\obs-studio
C;\ProgramData\obs-studio\shader-cache
```

I do not get the `obs-studio-hook` folder.

**Comment by niresh1234** [comment]
@RytoEX see this (also included content inside C:\ProgramData\obs-studio-hook folder)

[obs-studio-hook.zip](https://github.com/obsproject/obs-studio/files/15019172/obs-studio-hook.zip)

![Untitled-1-01](https://github.com/obsproject/obs-studio/assets/167388310/37a89f46-1bd4-40cf-a76d-4098acd64a3a)

**Comment by Fenrirthviti** [comment]
Are you absolutely certain this was placed by a portable install? Those files are expected from a traditional installation.

**Comment by niresh1234** [comment]
@Fenrirthviti yes, portable install..... using zip + obs_portable_mode.txt

**Comment by Fenrirthviti** [comment]
Thanks, confirmed in sandbox as well that the graphics hook is installed.

I'm not entirely sure this is something we can avoid, as it's required for the functionality of the game capture feature on windows. Potentially there could be a warning on first launch that certain files have to be installed to a shared user location for OBS to function properly, but I feel that just leads to more user confusion than anything else.

**Comment by niresh1234** [comment]
@Fenrirthviti if I delete C:\ProgramData\obs-studio-hook folder
and
run OBS, the folder created again.

**Comment by Fenrirthviti** [comment]
Yes, confirmed that to be the case. As mentioned above, this is currently required for OBS to function, so I'm not sure what the best solution here is. I don't believe that with the way game capture works we can make it fully portable. I'm not a fan of silently breaking functionality, and users don't read popups, so it's a bit of an awkward situation. We'll discuss internally.

**Comment by niresh1234** [comment]
@Fenrirthviti why not simply include this file in obs studio main\data folder?

**Comment by Fenrirthviti** [comment]
Because the game capture currently expects it to be there, as well as other permissions and security issues around how the game capture hook works. Putting in OBS's directory and loading it from an arbitrary location could potentially break capture with many games.

**Comment by lsdxfs** [comment]
I'm having the same problem, and I personally think it's better to be fully portable
